kinto/config/kinto.tpl

```diff
@@ -78,9 +159,80 @@ multiauth.policies = basicauth
 # fxa-oauth.relier.enabled = true
 # fxa-oauth.webapp.authorized_domains = *
 
+# Plugins
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#plugins
+# https://github.com/uralbash/awesome-pyramid
+kinto.includes = kinto.plugins.default_bucket
+#                kinto.plugins.admin
+#                kinto.plugins.accounts
+#                kinto.plugins.history
+#                kinto.plugins.quotas
```

Comment:
This plugins section is often used. I would suggest to move it after *Feature settings*